### PR TITLE
Fix debian systemV init script status command #552

### PR DESCRIPTION
**Pull request by kardapoltsev**

### CODE REVIEW

File: src/main/resources/com/typesafe/sbt/packager/archetypes/java_server/systemloader/systemv/start-debian-template [hunk 1]
```diff
@@ -70,22 +70,7 @@ case "$1" in
...
-      ;;
-    3)
-      echo "${{app_name}} is not running"
-      ;;
-    *)
-      echo "Unable to determine ${{app_name}} status"
-      ;;
-  esac
-  exit $?
+    status_of_proc -p "$PIDFILE" "$RUN_CMD" ${{app_name}} && exit 0 || exit $?
```
Comment (muuki88) [inline src/main/resources/com/typesafe/sbt/packager/archetypes/java_server/systemloader/systemv/start-debian-template]: LGTM. `status_of_proc` is available for all debian systems?

**Comment by kardapoltsev** [comment]
It's sourced from `/lib/lsb/init-functions`.

**Comment by muuki88** [comment]
Yep, but since when? E.g. like 12.04 LTS or earlier/later?

**Comment by kardapoltsev** [comment]
From https://wiki.ubuntu.com/InitScriptStatusActions:
> DustinKirkland worked with Debian to provide a method,  status_of_proc() , in /lib/lsb/init-functions, in the lsb-base library package. This function is fully integrated in Intrepid as of lsb-3.2-14.

Oldest available lsb package at https://launchpad.net/ubuntu/+source/lsb is 4.0

**Comment by muuki88** [comment]
Ah nice. This should be enough xD

**Pull request merged**